pom.xml

```diff
@@ -1,142 +1,151 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.synopsys.blackduck</groupId>
-  <artifactId>rapid-scan-maven-plugin</artifactId>
-  <packaging>maven-plugin</packaging>
-  <version>1.0-SNAPSHOT</version>
+    <groupId>com.synopsys.blackducksoftware</groupId>
+    <artifactId>rapid-scan-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <version>1.0.1-SNAPSHOT</version>
 
-  <name>Rapid Scan Maven Plugin</name>
-
-  <scm>
-    <connection>scm:git:https://github.com/blackducksoftware/rapid-scan-maven-plugin.git</connection>
-    <developerConnection>scm:git:https://github.com/blackducksoftware/rapid-scan-maven-plugin.git</developerConnection>
+    <name>Rapid Scan Maven Plugin</name>
+    <description>Add a description</description>
     <url>https://github.com/blackducksoftware/rapid-scan-maven-plugin</url>
-  </scm>
+    <developers>
+        <developer>
+            <name>Change Me</name>
+            <email>changeme@synopsys.com</email>
+            <organization>Synopsys</organization>
+            <organizationUrl>https://www.synopsys.com/</organizationUrl>
+        </developer>
+    </developers>
```

Comment:
@StevePitschke 
@sig-rlloyd 

```
<developers>
</developers>
``` 
is needed but I am not sure what to add here. Please let me know.

Reply:
You can put me and yourself iin there.